compat/src-deprecated/functions.php

```diff
@@ -839,3 +839,166 @@ function notification_update_setting($setting, $value) {
 
 	return notificationUpdateSetting($setting, $value);
 }
+
+
+/**
+ * Adapts Notification object
+ * Default adapters are: WordPress || JSON
+ *
+ * @param string $adapterName Adapter class name.
+ * @param \BracketSpace\Notification\Core\Notification $notification Notification object.
+ * @return \BracketSpace\Notification\Interfaces\Adaptable
+ * @throws \Exception If adapter wasn't found.
+ * @since  6.0.0
+ * @deprecated [Next]
+ */
+function notificationAdapt($adapterName, \BracketSpace\Notification\Core\Notification $notification) {
+	_deprecated_function( __FUNCTION__, '[Next]', 'BracketSpace\\Notification\\adapt()');
```

Comment:
Comment's doesn't match now. Check other functions as well
```suggestion
	_deprecated_function( __FUNCTION__, '[Next]', 'BracketSpace\\Notification\\adaptNotification()');
```